ngraph/frontend/paddlepaddle/src/frontend.cpp

```diff
@@ -138,14 +138,16 @@ std::shared_ptr<Function>
                 const auto& node = pdpd::make_ng_node(nodes_dict, op_place, CREATORS_MAP);
                 if (node) {
                     // set layer name by the name of first output var
-                    const auto& first_output_var = op_place->getOutputPorts().begin()->second->getTargetTensorPDPD(0)->getDesc();
+                    const auto& first_output_var = op_place->getOutputPorts().begin()->second[0]->getTargetTensorPDPD()->getDesc();
                     node->set_friendly_name(first_output_var->name());
 
                     std::cerr << "Named with " << node->get_friendly_name() << "\n";
-                    for (const auto &name_to_port : op_place->getOutputPorts()) {
-                        for (size_t idx = 0; idx < name_to_port.second->getTargetTensors().size(); ++idx) {
-                            const auto& var = name_to_port.second->getTargetTensorPDPD(idx)->getDesc();
+                    for (const auto& name_to_ports : op_place->getOutputPorts()) {
+                        int idx = 0;
```

Comment:
maybe size_t is better then 'int' here

Reply:
already fixed in https://github.com/nosovmik/openvino/pull/11